package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
```

Comment:
I used 4 spaces in every package.json (to follow the convention we use in PHP), I think we should stick with that. WDYT?

Reply:
Yeah let's go for it!
IMO the best things would to run Prettier on this file too. 

Stay tuned for a next PR... 👀